bot_services/answer_service.py

```diff
@@ -0,0 +1,15 @@
+from pprint import pprint
```

Comment:
In general use the built-in "print" function to print to STDOUT. pprint is for printing lists, dictionary, JSON, etc more prettily (pprint stands for pretty print)